src/java.desktop/share/classes/javax/imageio/spi/ServiceRegistry.java

```diff
@@ -78,7 +78,7 @@
  * proxy for the heavyweight service.
  *
  * <p> An application may customize the contents of a registry as it
- * sees fit, so long as it has the appropriate runtime permission.
+ * sees fit.
  *
```

Comment:
Removing the above text is not related to finalization, but the text is meaningless since the SecurityManager was permanently disabled, so I will take this opportunity to remove it.